docs/csharp/programming-guide/types/boxing-and-unboxing.md

```diff
@@ -55,7 +55,7 @@ It is also possible to perform the boxing explicitly as in the following example
 
 This example converts an integer variable `i` to an object `o` by using boxing. Then, the value stored in the variable `i` is changed from `123` to `456`. The example shows that the original value type and the boxed object use separate memory locations, and therefore can store different values.
 
-## Example
+## Example 1
```

Comment:
@gewarren This is an example of what I meant. This example is under `## Boxing` heading (it should probably have been H3 to indicate its a sub-heading - but that's not the point). The second example is under `## Unboxing` which is an example for something *different*. I'm not sure if my point is valid, but that's how I think about numbered examples (that they are different example for *one specific* concept). Imagine if this was rewritten in proper structure, that would be:

```md
## Unboxing

Definition of unboxing here.

### Example 1

## Boxing

Definition

### Example 2
```

However, `Example 2` is the first example of Unboxing.

Reply:
I see what you mean. However, that's beyond the scope of this PR. As you say, these Example headings should really be H2s, as should the Description H2. This is the current in-article TOC which is definitely not great.

![image](https://user-images.githubusercontent.com/24882762/127020438-00805306-92b9-422c-86a0-3c49ab9a8742.png)

Reply:
If we make the Example headings H3 (which makes sense to me), they shouldn't need numbers to differentiate them. Do we get duplicate heading warnings for H3 headings?

Reply:
Let me try it. The bookmarks still wouldn't function properly even if there are no warnings.

Reply:
No warnings for duplicate H3s.